### PR TITLE
Fix typos under torch/ao directory

### PR DESCRIPTION
**Pull request by kiszk**
This PR fixes typos in comments and messages of `.py` files under `torch/ao` directory

### CODE REVIEW

File: torch/ao/pruning/_experimental/data_sparsifier/data_norm_sparsifier.py [hunk 1]
```diff
@@ -128,7 +128,7 @@ def update_mask(self, name, data, sparsity_level,
         else:
             data_norm = (data * data).squeeze()  # square every element for L2
 
-        if len(data_norm.shape) > 2:  # only supports 2 dimenstional data at the moment
+        if len(data_norm.shape) > 2:  # only supports 2 dimensitional data at the moment
```
Comment (janeyx99) [inline torch/ao/pruning/_experimental/data_sparsifier/data_norm_sparsifier.py:131]:
```suggestion
        if len(data_norm.shape) > 2:  # only supports 2 dimensional data at the moment
```

**Review by janeyx99 — CHANGES_REQUESTED** [review]
small nit, overall fine!

**Comment by kiszk** [comment]
@janeyx99 Thank you for your comment. I addressed it.

**Review by janeyx99 — APPROVED** [review]
👍

**Review by kit1980 — APPROVED** [review]

**Pull request closed without merging**